install/kubernetes/helm/istio/charts/gateways/values.yaml

```diff
@@ -25,7 +25,7 @@ istio-ingressgateway:
   cpu:
     targetAverageUtilization: 80
   loadBalancerIP: ""
-  loadBalancerSourceRanges: {}
+  loadBalancerSourceRanges: []
```

Comment:
I think here should be a map but not array, I did not encounter any issue when running the helm chart. @tim-beatson @ymesika

Reply:
It is a list field but I think that as long as it's empty it doesn't matter if its `{}` or `[]`.
Maybe you should align this one with how we define it elsewhere. I.e. simply `loadBalancerSourceRanges:`. nit.

Reply:
@ymesika +1 to clear this and leave it as empty, this is the best practise to handle such values.